<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeuApp</class>
 <widget class="QMainWindow" name="MeuApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>App_Futebol</string>
  </property>
  <property name="toolTipDuration">
   <number>12</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="toolTipDuration">
    <number>12</number>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>501</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>12</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #f8f9fa;
    color: #343a40;

    padding: 10px;
    font-size: 14px;
}</string>
    </property>
    <widget class="QPushButton" name="btnFechar">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>10</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #007bff;

    padding: 10px 20px;
    font-size: 16px;
    margin-top: 10px;
}

QPushButton:hove {
    background-color: #e5e5e5;
    color: #0056b3;
}

QPushButton:presse {
    background-color: #c6c6c6;
    color: #004080;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../../Users/brunomendes/Downloads/icons/fechar3.png</normaloff>../../../../../../Users/brunomendes/Downloads/icons/fechar3.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>861</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>12</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QStackedWidget {
    background-color: #f8f9fa;
    color: #343a40;
    border: 2px solid #6c757d;
    padding: 10px;
    font-size: 14px;
}

QStackedWidget:focus {
    border: 2px solid #007bff;
}
</string>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="home">
      <widget class="QTextEdit" name="btnResumoHome">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>831</width>
         <height>341</height>
        </rect>
       </property>
       <property name="toolTipDuration">
        <number>12</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit {
    background-color: #f8f9fa;
    color: #343a40;
    border: 2px solid #6c757d;
    border-radius: 20px;
    padding: 8px;
    font-size: 14px;
}</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Historia">
      <property name="toolTipDuration">
       <number>12</number>
      </property>
      <widget class="QTextEdit" name="btnResumoHistoria">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>831</width>
         <height>341</height>
        </rect>
       </property>
       <property name="toolTipDuration">
        <number>12</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit {
    background-color: #f8f9fa;
    color: #343a40;
    border: 2px solid #6c757d;
    border-radius: 20px;
    padding: 8px;
    font-size: 14px;
}</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Jogadores">
      <widget class="QTextEdit" name="btnResumoJogadores">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>831</width>
         <height>341</height>
        </rect>
       </property>
       <property name="toolTipDuration">
        <number>12</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit {
    background-color: #f8f9fa;
    color: #343a40;
    border: 2px solid #6c757d;
    border-radius: 20px;
    padding: 8px;
    font-size: 14px;
}</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Noticias">
      <property name="toolTipDuration">
       <number>12</number>
      </property>
      <widget class="QLabel" name="btnTitulo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>821</width>
         <height>81</height>
        </rect>
       </property>
       <property name="toolTipDuration">
        <number>12</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: #f8f9fa;
    color: #343a40;
    border: 2px solid #6c757d;
    border-radius: 20px;
    padding: 8px; 
    font-size: 18px;
    margin-bottom: 20px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLink">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>290</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="toolTipDuration">
        <number>12</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #007bff;
    color: white;
    border: 2px solid #007bff;
    border-radius: 15px;
    padding: 10px 20px;
    font-size: 16px;
    margin-top: 10px;
}

QPushButton:hover {
    background-color: #0056b3;
}

QPushButton:pressed {
    background-color: #004080;
}
</string>
       </property>
       <property name="text">
        <string>Acessar Site</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="btnResumo">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>571</width>
         <height>211</height>
        </rect>
       </property>
       <property name="toolTipDuration">
        <number>12</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit {
    background-color: #f8f9fa;
    color: #343a40;
    border: 2px solid #6c757d;
    border-radius: 20px;
    padding: 8px;
    font-size: 14px;
}</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLinkNoticia">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>290</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="toolTipDuration">
        <number>12</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #007bff;
    color: white;
    border: 2px solid #007bff;
    border-radius: 15px;
    padding: 10px 20px;
    font-size: 16px;
    margin-top: 10px;
}

QPushButton:hover {
    background-color: #0056b3;
}

QPushButton:pressed {
    background-color: #004080;
}
</string>
       </property>
       <property name="text">
        <string>Acessar Notícias</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnProximo">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>130</y>
         <width>101</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #007bff;
    color: white;
    border: 2px solid #007bff;
    border-radius: 15px;
    padding: 10px 20px;
    font-size: 16px;
    margin-top: 10px;
}

QPushButton:hover {
    background-color: #0056b3;
}

QPushButton:pressed {
    background-color: #004080;
}
</string>
       </property>
       <property name="text">
        <string>Próximo</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAnterior">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #007bff;
    color: white;
    border: 2px solid #007bff;
    border-radius: 15px;
    padding: 10px 20px;
    font-size: 16px;
    margin-top: 10px;
}

QPushButton:hover {
    background-color: #0056b3;
}

QPushButton:pressed {
    background-color: #004080;
}
</string>
       </property>
       <property name="text">
        <string>Anterior</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #007bff;
    color: white;
    border: 2px solid #007bff;
    border-radius: 15px;
    padding: 10px 20px;
    font-size: 16px;
    margin-top: 10px;
}

QToolButton:hover {
    background-color: #0056b3;
}

QToolButton:pressed {
    background-color: #004080;
}
</string>
     </property>
     <property name="text">
      <string>Notícias</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>10</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #007bff;
    color: white;
    border: 2px solid #007bff;
    border-radius: 15px;
    padding: 10px 20px;
    font-size: 16px;
    margin-top: 10px;
}

QToolButton:hover {
    background-color: #0056b3;
}

QToolButton:pressed {
    background-color: #004080;
}
</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>187</x>
       <y>10</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #007bff;
    color: white;
    border: 2px solid #007bff;
    border-radius: 15px;
    padding: 10px 20px;
    font-size: 16px;
    margin-top: 10px;
}

QToolButton:hover {
    background-color: #0056b3;
}

QToolButton:pressed {
    background-color: #004080;
}
</string>
     </property>
     <property name="text">
      <string>História</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>361</x>
       <y>10</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #007bff;
    color: white;
    border: 2px solid #007bff;
    border-radius: 15px;
    padding: 10px 20px;
    font-size: 16px;
    margin-top: 10px;
}

QToolButton:hover {
    background-color: #0056b3;
}

QToolButton:pressed {
    background-color: #004080;
}
</string>
     </property>
     <property name="text">
      <string>Jogadores</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
